src/containerapp/azext_containerapp/_utils.py

```diff
@@ -2083,3 +2083,71 @@ def parse_oryx_mariner_tag(tag: str) -> OryxMarinerRunImgTagProperty:
     else:
         tag_obj = None
     return tag_obj
+
+
+def load_yaml_file(file_name):
```

Comment:
If these methods are only used for decorator, could you saparate them to _decorator_utils?